lib/urcli-money.js

```diff
@@ -66,8 +66,17 @@ function definePeriods () {
       }
       const yearAndMonth = arg < 9 ? `${year}-0${arg}` : `${year}-${arg}`
       let start = moment(yearAndMonth).utc()
-      let end = moment(yearAndMonth).utc().add(1, 'M')
-      periods.push([start, end])
+      // Check if month asked is the current month
+      let currentMonth = moment.utc().add(1, 'M').month()
+      if (currentMonth == arg) {
```

Comment:
In JavaScript "strict mode" we use "===" not "==". I know it's weird coming from Python, but that's the way it is 😄 . This is enforced by the Standard styleguide, which we're using in this project.